resume-buddy/src/main/java/com/google/sps/servlets/RevieweeDataServlet.java

```diff
@@ -32,15 +44,65 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String fname = ServletHelpers.getParameter(request, "fname", "");
     String lname = ServletHelpers.getParameter(request, "lname", "");
     String email = ServletHelpers.getParameter(request, "email", "");
-    reviewee = new Reviewee(fname, lname, email);
+    String resumeURL = getUploadedFileUrl(request, "resume");
+    reviewee = new Reviewee(fname, lname, email, resumeURL);
+
     Entity revieweeEntity = new Entity("Reviewee");
     revieweeEntity.setProperty("first-name", fname);
     revieweeEntity.setProperty("last-name", lname);
     revieweeEntity.setProperty("email", email);
+    revieweeEntity.setProperty("resumeURL", resumeURL);
 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     datastore.put(revieweeEntity);
 
     response.sendRedirect("resume-review.html");
   }
+
+  /**
+   * @return the request parameter, or the default value if the parameter was not specified by the
+   *     client
+   */
+  private String getParameter(HttpServletRequest request, String name, String defaultValue) {
+    String value = request.getParameter(name);
+    return (value == null) ? defaultValue : value;
+  }
+
+  /** Returns a URL that points to the uploaded file, or null if the user didn't upload a file. */
+  private String getUploadedFileUrl(HttpServletRequest request, String formInputElementName) {
+    BlobstoreService blobstoreService = BlobstoreServiceFactory.getBlobstoreService();
+    Map<String, List<BlobKey>> blobs = blobstoreService.getUploads(request);
+    List<BlobKey> blobKeys = blobs.get("image");
```

Comment:
You should be looking for the "resume" key instead of the "image" key.

Reply:
Fixed!